WcaOnRails/db/migrate/20160610191605_create_competition_tabs.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCompetitionTabs < ActiveRecord::Migration
+  def change
+    create_table :competition_tabs do |t|
+      t.string :competition_id
```

Comment:
Sorry I misunderstood you, I guess you meant updating foreign keys and yeah there's a method for that. Really good catch, thanks!